### DISCUSSION TITLE
Invalid compression options with Cassandra 5

### DISCUSSION THREAD

**Discussion by To-om** [JanusGraph/janusgraph#4688] [General]
Cassandra version 5 has been released. Early tests seem to show that only table creation is causing issues. In the compression options, `sstable_compression` and `chunk_length_kb` are used [here](https://github.com/JanusGraph/janusgraph/blob/73232fe09a4dff2d447c89b5a63b72a890d3342f/janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java#L334), but they have been deprecated since [Cassandra 3](https://github.com/apache/cassandra/blob/219eea33b6102ceac4832368444878b64ff2a63a/NEWS.txt#L1419) and removed in [Cassandra 5](https://github.com/apache/cassandra/blob/219eea33b6102ceac4832368444878b64ff2a63a/NEWS.txt#L335). They have been replaced by `class` and `chunk_length_in_kb`.

Even though support for Cassandra version 5 is not official yet, replacing these options would make testing this new version easier. Would you like me to create a PR for this?

**Comment by cdegroc** [comment]
Yes, if you could submit a PR that'd be great!

There is information on contributing [here](https://github.com/JanusGraph/legal/#contributing), in particular the requirement that contributors sign the [Contributor License Agreement (CLA)](https://github.com/JanusGraph/legal).
